### ISSUE TITLE
External SCSS

### ISSUE THREAD

**Issue by stinovlas** [vuejs/vue-jest#178]
After upgrading jest to 24.8.0, I got a bunch of warnings regarding SCSS in my Vue components. I mitigated most of them by setting `moduleNameMapper`, as suggested in the warning messages. However, some of them didn't disappear. It seems that linking external scss module like this is source of the problem:

```html
<style lang="scss" module src="../MyCustomStyles.module.scss"/>
```

The warning goes as this:

```
[vue-jest]: There was an error rendering the SCSS in /home/user/path/to/my/project/components/MyComponent.vue. SCSS is fully supported by vue-jest. Still some features might throw errors. Webpack aliases are a common cause of errors. If you use Webpack aliases, please use jest's suggested way via moduleNameMapper which is supported.


[vue-jest]: Error while compiling styles: Error: No input specified: provide a file name or a source string to process
```

I'm using `vue-jest@3.0.4`, `jest@24.8.0` and `vue@2.6.10`.

If anyone has any idea what am I doing wrong, I'd be glad for help.

**Comment by stinovlas** [comment]
Sorry to bother you, the issue disappeared itself. Probably some cache issues.

**Comment by eddyerburgh** [comment]
First, have you tried vue-jest@beta?

Second, you can turn off style warnings in the Jest configuration options:

  ```json
  {
    "jest": {
      "globals": {
        "vue-jest": {
          "hideStyleWarn": true
        }
      }
    }
  }
```

**Comment by stinovlas** [comment]
The problem seems to be resolved in `v4.0.0-beta.2`. There is even external css support, which is a big bonus, thanks for that! Do you have any idea when version 4.0.0 might reach a stable (non-beta) phase?

I generally prefer not to silence warnings, because they may be useful. These particular warnings were not, because they were not reflecting the reality, but otherwise, I like relevant warnings and want to keep them showing up.

**Comment by stinovlas** [comment]
Hello again! I've been waiting for stable release that would fix this issue for some time. There has been version `v4.0.0-rc.0` with comment about releasing `v4.0.0` before the end of Sep 2020 in release notes, but that didn't happen. Is there any chance that we'll see stable `v4.0.0` version anytime soon?
